contracts/src/SuccinctGateway.sol

```diff
@@ -29,8 +35,10 @@ contract SuccinctGateway is ISuccinctGateway, FunctionRegistry, TimelockedUpgrad
     /// @notice A flag that indicates whether the contract is currently making a callback.
     bool public isCallback;
 
+    mapping(bytes32 => WhitelistStatus) public whitelistStatus;
```

Comment:
Need a way for function owners to update this, otherwise always stuck on default.